README.md

```diff
@@ -7,14 +7,20 @@
 
 Plugin that generate sprites from your stylesheets (using [spritesmith](https://github.com/Ensighten/spritesmith)) and then updates image references.
 
-## Getting started
+## Getting Started
 
 If you haven't used [gulp](http://gulpjs.com) before, be sure to check out the [Getting Started](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md) guide.
 
-Install with [npm](https://npmjs.org/package/gulp-sprite-generator)
+Install with [npm](https://npmjs.org/package/gulp-sprite-generator):
 
+```sh
+$ npm install --save-dev gulp-sprite-generator
 ```
-npm install --save-dev gulp-sprite-generator
+
+You also should install a spritesmith engine:
```

Comment:
Why user should install it?
Spritesmith will install it via it's package.json, [isnt it](https://github.com/Ensighten/spritesmith/blob/master/package.json#L35)?